Users/Install.rst

```diff
@@ -285,6 +285,13 @@ For x86 python or for x64 python:
 
 Replace "XXX" with the ClangSharp version you received from nuget.
 
+Shell-autocompletion support
+----------------------------
+
+If you are a bash/zsh user , checkout the
+`guide <http://docs.coala.io/en/latest/Users/Shell_Autocompletion.html>`
+for setting-up tab-completion for coala arguements and bear names.
```

Comment:
i mean to say that do i need to do separate commit apart from last commit or just amend my last commit?